src/Components/Artwork/Details/PartnerOfferLine.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Flex, Text } from "@artsy/palette"
+import { useTimer } from "Utils/Hooks/useTimer"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { PartnerOfferLineQuery } from "__generated__/PartnerOfferLineQuery.graphql"
+import { PartnerOfferLine_artwork$data } from "__generated__/PartnerOfferLine_artwork.graphql"
+
+interface PartnerOfferLineProps {
+  artwork: PartnerOfferLine_artwork$data
+}
+
+const ActivePartnerOfferTimer: React.FC<PartnerOfferLineProps> = ({
+  artwork: { collectorSignals },
+}) => {
+  const SEPARATOR = <>&nbsp;</>
+  const { endAt } = collectorSignals?.partnerOffer ?? {}
+  const { time } = useTimer(endAt ?? "")
+  const { days, hours } = time
+
+  return (
+    <Text
+      variant="sm-display"
+      lineHeight="22px"
+      color="red100"
+      alignSelf="flex-start"
+    >
+      Offer Expires {SEPARATOR}
+      {Number(days)}d{SEPARATOR}
+      {Number(hours)}h{SEPARATOR}
+    </Text>
+  )
+}
+
+const PartnerOfferLine: React.FC<PartnerOfferLineProps> = ({ artwork }) => {
```

Comment:
This is the combined fifth line that's now fetched and rendered on the client side.